core/src/test/java/io/grpc/PickFirstLoadBalancerTest.java

```diff
@@ -71,6 +73,7 @@
 
   private static final Attributes.Key<String> FOO = Attributes.Key.of("foo");
   private Attributes affinity = Attributes.newBuilder().set(FOO, "bar").build();
+  private static final PickSubchannelArgs EMPTY_ARGS = mock(PickSubchannelArgs.class);
```

Comment:
Good points. Renamed it to `mockArgs`, which stick nicely with other fields and the convention we have.